.github/workflows/tests.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Should we run it for every PR?

Reply:
"every PR targeting master" yes I believe

Reply:
Enabled it for all PRs